mm2src/coins/eth.rs

```diff
@@ -5142,14 +5117,14 @@ impl EthCoin {
     }
 
     /// Get gas base fee and suggest priority tip fees for the next block (see EIP-1559)
-    pub async fn get_eip1559_gas_fee(&self) -> Web3RpcResult<FeePerGasEstimated> {
+    pub async fn get_eip1559_gas_fee(&self, use_simple: bool) -> Web3RpcResult<FeePerGasEstimated> {
```

Comment:
Why do we need this use_simple param and not just get the estimator config from the single place?
(It was supposed btw that the simple estimator won't be used and we always connect a provider for gas fee data)

Reply:
Well, I see it's kind of an override param and if it's false the main config used